File: Intro/php/isPrime.php
<?php
declare(strict_types=1);

function is_prime(int $n): bool {
    if ($n < 2) return false;
    for ($i = 2; $i * $i <= $n; $i++) {
        if ($n % $i === 0) return false;
    }
    return true;
}

File: Intro/csharp/IsPrime.cs
namespace IntroSolutions;

public static class IsPrime
{
    public static bool Solve(int n)
    {
        if (n < 2)
        {
            return false;
        }

        for (int i = 2; i * i <= n; i++)
        {
            if (n % i == 0)
            {
                return false;
            }
        }

        return true;
    }
}
